pkg/state/state.go

```diff
@@ -464,6 +464,18 @@ func newStateManager(dataDir string, params StateParams, settings *settings.Bloc
 	return state, nil
 }
 
+func (s *stateManager) GetByteTree(recipient proto.Recipient) (proto.Script, error) {
+	if recipient.Address != nil {
```

Comment:
Better not just check that Recipient is an address, but lookup for an address here by alias from Recipient.